src/Common/gui/MainWindow.cpp

```diff
@@ -2087,7 +2087,7 @@ void MainWindow::timerEvent(QTimerEvent *)
                    auto memmoryUsed = performanceMonitor->getMemmoryUsed();
                    auto batteryUsed = performanceMonitor->getBatteryUsed();
 
-                   bool showMemmory = memmoryUsed > 60; //memory meter only active if memory usage is <60%
```

Comment:
Hi @arnaudlecam I think the comment is wrong here, not the operator. The idea is to show the memory used as a warning when it exceeds 60% . It was fixed before, but when you have lot's of memory available it is useless Sometimes other software or big plugin libraries consume a lot of ram, so this is a quick indicator that something is wrong.